tests/ui/ref_option_ref.rs

```diff
@@ -0,0 +1,42 @@
+#![allow(unused)]
```

Comment:
```suggestion
// run-rustfix
#![allow(unused)]
```

Since this should be auto-fixable, we should also test this.

Reply:
> Since this should be auto-fixable, we should also test this. Remember to also update the reference files after this change. (That should be the last thing before merging.)

I thought about that but that would imply also fixing usages of every variables declared with this type.
Should this lint also suggest these fixes ?

Reply:
No, I think that would be overkill and really hard to create a working suggestion there. But you can set the applicability level to `MaybeIncorrect`. Please also leave a comment in the test, why this is not `run-rustfix`